pipeline/restart_node.sh

```diff
@@ -413,7 +414,7 @@ upgrade_binaries() {
 	'
 }
 
-unset -v logsize zerologsize
+unset -v logsize zerologsize do_zerologsize
```

Comment:
do you still need this variable do_zerologsize?

Reply:
removed